unittests/savanna_cluster.hpp

```diff
@@ -66,11 +67,13 @@ namespace savanna_cluster {
 
       uint32_t lib_num() const { return lib_block->block_num(); }
 
-      uint32_t head_num() const { return control->fork_db_head_block_num(); }
+      uint32_t forkdb_head_num() const { return control->fork_db_head_block_num(); }
+
+      signed_block_ptr forkdb_head() const { return control->fork_db_head_block(); }
```

Comment:
Doesn't look like this is used. Also `fork_db_head_block()` only used here.

Reply:
right, removed.